planner/core/issuetest/BUILD.bazel

```diff
@@ -9,7 +9,7 @@ go_test(
     ],
     flaky = True,
     race = "on",
-    shard_count = 4,
+    shard_count = 5,
```

Comment:
why this change?

Reply:
It seems that the shard will increase as the test increase. The maximum number of the shard is 50.